### DISCUSSION TITLE
[Bug] Notifications hide new activity

### DISCUSSION THREAD

**Discussion by SharakPL** [community/community#42986] [Mobile]
1. I get new notification about a PR/issue etc.
2. PR/issue gets new activity eg. new commits, new comments etc.
3. Then I open the notification mentioned in 1
4. It shows everything from the activity that generated the notification but after few seconds all new activity gets hidden and only this one remains.

https://user-images.githubusercontent.com/5007456/210133605-0efa57b9-dd74-4880-8cb5-7df29a2e3df3.mp4

**Expected behaviour:**
It shouldn't hide anything.

**Comment by eliperkins** [comment]
Hey @SharakPL , thanks for sharing this bug report. This seems like a bug on our end, so I've put your feedback here into an issue in our internal tracking, and followed up with the engineering team to get some eyes on this.

I'll let you know when the bug is fixed! Thanks for your patience!

**Reply by SharakPL** [reply]
Thanks @eliperkins

**Comment by stevepopovich** [comment]
Hi @SharakPL, are you seeing this consistently? 

We are having trouble reproducing, but have some changes in the next release that might help fix this. 

Thanks for filing the issue!

**Reply by SharakPL** [reply]
Yes, it is consistent. Though behaviour is different based on type of new content.
For new comments it's like shown in the first video.
When it's `<user> approved these changes` type then notfication opens just this one line and I need to hit **Back** to view whole conversation. I honestly don't know what's the point of showing this line, the same can be seen in notification already so it's useless to open it once again as full screen notification. It also should just open the conversation at the first unread part and allow to just read on next part.

https://user-images.githubusercontent.com/5007456/211929035-57242aed-a6ca-491d-9949-be7ec4dca57d.mp4

When it's `<user> pushed 1 commit` type it goes straight to Review for whole PR for some reason, not that one commit.

It looks like the comment type tries to behave like the rest: show just this comment on separate page, but instead it opens whole conversation and then tries to hide everything else besides this one comment.

**Reply by stevepopovich** [reply]
Hi @SharakPL, are you still seeing this issue?
